src/Hosting/Hosting/src/Internal/HostingEventSource.cs

```diff
@@ -90,23 +94,23 @@ protected override void OnEventCommand(EventCommandEventArgs command)
             // This is the convention for initializing counters in the RuntimeEventSource (lazily on the first enable command).
             // They aren't disabled afterwards...
 
-            _requestsPerSecondCounter ??= new IncrementingPollingCounter("requests-per-second", this, () => Volatile.Read(ref _totalRequests))
+            _requestsPerSecondCounter ??= new IncrementingPollingCounter("requests-per-second", this, () => TotalRequests)
```

Comment:
What's the equivalent of this counter in metrics?

Reply:
Are you saying that this counter isn't relevant with metrics? It's the job of infrastructure that collects and presents information to display this using `total-requests`? e.g. dotnet-counters, Open Telemetry